### ISSUE TITLE
Multiple Printers? 

### ISSUE THREAD

**Issue by bamboo-eater** [bigtreetech/BTT-Pi#6]
Is it possible to connect more than 1 printer to the BTT Pi?  If so are there any documentation on how to do this. Thanks

**Comment by flavio-costa** [comment]
yes it is possible, I uploaded klipper using kiauh, there is an option to choose the number of machines to install, it is called instances)
